climada/entity/impact_funcs/wildfire.py

```diff
@@ -0,0 +1,59 @@
+"""
+This file is part of CLIMADA.
+Copyright (C) 2017 ETH Zurich, CLIMADA contributors listed in AUTHORS.
+CLIMADA is free software: you can redistribute it and/or modify it under the
+terms of the GNU Lesser General Public License as published by the Free
+Software Foundation, version 3.
+CLIMADA is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more details.
+You should have received a copy of the GNU Lesser General Public License along
+with CLIMADA. If not, see <https://www.gnu.org/licenses/>.
+---
+Define impact functions for BushFires.
+"""
+
+__all__ = ['IFWildFire']
+
+import logging
+import numpy as np
+
+from climada.entity.impact_funcs.base import ImpactFunc
+
+LOGGER = logging.getLogger(__name__)
+
+class IFBushfire(ImpactFunc):
+    """Impact function for bushfire."""
+
+    def __init__(self):
+        ImpactFunc.__init__(self)
+        self.haz_type = 'WF'
```

Comment:
```suggestion
        self.haz_type = 'WF'
        self.id = 1
        self.intensity_unit = "K"
```

Reply:
Where is the `id` used? Why is it always `1`? Is it possible to rename it to something better searchable, like `wildfire_impf_id`?

Reply:
fully re-written the impact_funcs -> took id as an input as in tc_impact_funcs